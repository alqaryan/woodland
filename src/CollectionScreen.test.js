const CollectionScreen = require('./CollectionScreen.js');
describe('CollectionScreen', function() {
    it('should be exist', function() {
      expect('CollectionScreen').toBeDefined();
    });

    it('should be a function', function(){
      expect(typeof CollectionScreen).toBe("function");
    })
  });