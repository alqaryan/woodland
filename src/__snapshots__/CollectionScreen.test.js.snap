// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CollectionScreen renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#fff",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text>
    Collection
  </Text>
</View>
`;
